.gitignore

```diff
@@ -143,3 +143,5 @@ nbproject
 tests/e2e/screenshots
 tests/e2e/videos
 storybook-static
+*.swo
+*.swp
```

Comment:
Found the vim user ;)

Good QoL catch; thanks for adding these to `.gitignore` so we can keep 'em out of version control